features/dd-sdk-android-flags/api/apiSurface

```diff
@@ -29,6 +29,10 @@ interface com.datadog.android.flags.StateObservable
   fun getCurrentState(): com.datadog.android.flags.model.FlagsClientState
   fun addListener(FlagsStateListener)
   fun removeListener(FlagsStateListener)
+class com.datadog.android.flags._FlagsInternalProxy
+  constructor(FlagsClient)
+  fun getFlagAssignmentsSnapshot(): Map<String, com.datadog.android.flags.model.UnparsedFlag>?
```

Comment:
Minor: normally returning nullable collection should be avoided, it is better to return empty collection if there is nothing to return. This way there is no `null` case to handle at the call site.